inst/json/graph_prevalence_config.json

```diff
@@ -31,46 +35,34 @@
             "id": "llin",
             "name": "Pyrethroid LLIN only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "blue"}
         },
         {
             "id": "irs",
             "name": "IRS only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "purple"}
         },
         {
             "id": "irs-llin",
             "name": "Pyrethroid LLIN with IRS",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "blue"}
         },
         {
             "id": "llin-pbo",
             "name": "Pyrethroid-PBO LLIN only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "aquamarine"}
         },
         {
             "id": "irs-llin-pbo",
             "name": "Pyrethroid-PBO LLIN with IRS",
             "type": "lines",
-            "marker": {"color": "grey"}
-        },
-        {
-             "x": [12, 12],
```

Comment:
I took out this vertical line we were drawing at the one year mark pending decision from Ellie and Tom as to whether they want to keep it, and also because I eventually realised that was what was messing up the autorange! If we want to keep it we'll have to reinstate it with upper y value based on the data.